README.md

```diff
@@ -1,8 +1,8 @@
 # Oryx
 
-[![CI checks](https://github.com/spryker/oryx/actions/workflows/test.yml/badge.svg)](https://github.com/spryker/oryx/actions/workflows/test.yml)
-[![Oryx component dev env status](https://api.netlify.com/api/v1/badges/983ba7e1-0787-4036-a333-b1e01ef10fea/deploy-status)](https://app.netlify.com/sites/dev-oryx-components/deploys)
-[![Composable storefront dev env status](https://api.netlify.com/api/v1/badges/873ecabe-76b0-4493-af27-a0d150844cae/deploy-status)](https://app.netlify.com/sites/dev-composable-storefront/deploys)
+CI Checks | Oryx component dev env status | Composable storefront dev env status
+:-----: | :---: | :---:
+[![CI checks](https://github.com/spryker/oryx/actions/workflows/test.yml/badge.svg)](https://github.com/spryker/oryx/actions/workflows/test.yml) | [![Oryx component dev env status](https://api.netlify.com/api/v1/badges/983ba7e1-0787-4036-a333-b1e01ef10fea/deploy-status)](https://app.netlify.com/sites/dev-oryx-components/deploys) | [![Composable storefront dev env status](https://api.netlify.com/api/v1/badges/873ecabe-76b0-4493-af27-a0d150844cae/deploy-status)](https://app.netlify.com/sites/dev-composable-storefront/deploys)
```

Comment:
```suggestion
[![CI checks](https://github.com/oryx-frontend/oryx/actions/workflows/test.yml/badge.svg)](https://github.com/oryx-frontend/oryx/actions/workflows/test.yml) | [![Oryx component dev env status](https://api.netlify.com/api/v1/badges/983ba7e1-0787-4036-a333-b1e01ef10fea/deploy-status)](https://app.netlify.com/sites/dev-oryx-components/deploys) | [![Composable storefront dev env status](https://api.netlify.com/api/v1/badges/873ecabe-76b0-4493-af27-a0d150844cae/deploy-status)](https://app.netlify.com/sites/dev-composable-storefront/deploys)
```

Reply:
@PhilinTv Done. I've switched storybook and dev build for storefront for now. Later we will need production build and probably labs.